ER_Diagram.xml

```diff
@@ -0,0 +1 @@
+<mxfile type="device" userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.106 Safari/537.36" version="5.2.7.4" editor="www.draw.io"><diagram>5VtLc+I4EP41HEP5AQYfk8wkc9ndqUlN7e4ppWDFqGIsVjZ57K+flt0NfsjgYEOcGi7g1qvV3dL3qWVG7vXq9Vax9fIPGfBo5FjB68j9MnKcC9uy4EtL3nLJ1PNyQahEkIvsneBO/M9RiO3CjQh4UqqYShmlYl0WLmQc80VakjGl5Eu52qOMyqOuWYgj4oBacLdgEekxnu7kf4sgXebyuYPz0PJvXIRLGtv2/LzkgS2eQiU3MY44ctzH7JMXrxj1lY3sfgUzKimhG/1r9XrNI21KshKql6RvpNrIvVqmqwgebPiZFd80NMbJ7m8MU1E8xmnkwzX2N89bPLNoQx06XgRtrx4ldFEcyvtvoyeVFVwkmYMvoYI9Wb/uCuFXiN9ZLxsS/Ey4SkgKCm0LCrJ8yLI4EM9VUWvNLJNmPxMRh1D6F9uA37DLB/VR6ixZcr9i8RtUUPxZcIjz/RqA2KBEVVqJkix6uXa6jpJHEUXXMpIqK3S9a9+71PIkVfKJU0ksY2h9lSxZIF+wZahYICC2Cq2nzsyZXeJ073BIXXdvWGIgP3OVctxjMhGG6S2XK54qbRUs9WkDwv3Hm+Pzy24tO1RnWVjHHsoYbiDhtuvduoAfuDTMy2SKe0eXZeK1WiZXUj71tEySNYuNmkUi5hdkokw3Z59u3eJ7A+v+Hux4MKhzddvEejtRN7VTkUKzPlbiiRQs7l5F08GyPGjT9pPpXW2TysALFkqsUyG1aLi6rzcPkUiWvGjhTuqYpthNxZDHip9+ufWv+LuBeHhTOAbJBzaFBx7JOEzuU6lpL0DRWWbw0RBVmnW2gvqZ9hA5WI1wGWhZIwezbSJdSMJcv07CbDrxFUmYTRU7sTA8qZ3+sPKjtn4/7LjSmj+dAwFhSzigSOOKNvZvZS2t+mofa89WBn8qm+E0G093NavMsZOeXd0F2ZpB0b73wPtZLFTc+vVSG6Ba7XB4wMgTCAUZPc3u3S+J1OeWntDIw2M5pQQmBjQiWQmNSNgJjWYGNKoYnsfBpc5dwtMiYkkiFmXL8FeR/lP4/S/8tiBRCU8xqKOLCo+6dIfkPKCEZ5PlQBW5UQuqham+lKmQY7UcUesWLhhwarAfyRSPWCqey2qYbIojfJciWxXkQKua06Fn6iPXH5sV05i1nkglyk7TM/WUz7rWU+bk7cTb+d2UMj3S7+hc9HxWsvV70et5veP8TrmrAfl9SxHJ7+6xfrdnlZ4ovd6/3z1KWZ48B3gJW6TsK1l+QqajgekwJzNC/llZmvHoGLNVbyyt8xR7TkNkl1JZIkK76POlIQqZlOw4fp4ZnCbO0iWwrxBStTVf/J4pBLeC1BML8elgCqGPixwPsboX8G4mbVsk7wreBDol8M4MtocW29OxDZmOie/Mpq7nlEny1LfGjuXM/TlwpJkzQRbbEvnfAfLOZDKegy7+BIdro8b7CcDeyXqeP/Zt15q4kxlc9FeYSo/cwG3kBjXo7paqum25H9aGPS816BNdT6jmXgAfqtIFdGwDKR3ISY9JDH1dWbqa/MDsSjMkf77cyinyKPAaxTjfTV17PgeAxrz7WdIqpEsnhG5InhBim4/eRyI0bv1tEPo8x+s5nYHpkqbqlrbgOq8cruEdnJMB6LFJlQR0SNsFg4GUmYMh3437omt5gHxUMPiVVEvtdai2weBTQ+qIOj4QDOAbpqdL1da6Auy5jbzRrUSdW36FEX7kXR4davhWp4GrdUGadheLGX3r8fabrdaabT0k+utPIF0tDphtk0mDOD5fg2tLl1iDPej/kEDKYCP4BKp+k7BhtnhP7Bxk7EpIoD7rpYBXtYdBD29YnLJkIBendwtxcSOGoct3Kde/O1muwSBdWZ2FHM9MdxA90yTDxVPzNeWRNAkheLg0afuiU1eaBMmnnjgzPO7+VZJX3/1Tx/36Cw==</diagram></mxfile>
```

Comment:
Remove this file from your git, as it's not pertinent to your actual code. Upload a screenshot to imgur and link the image in your readme.md (like you did with the badges)